src/applications/financial-status-report/tests/e2e/fsr-5655-wizard.cypress.spec.js

```diff
@@ -13,7 +13,7 @@ Cypress.Commands.add('checkStorage', (key, expectedValue) => {
 describe('Financial Status Report (Wizard)', () => {
   before(() => {
     sessionStorage.setItem(WIZARD_STATUS, WIZARD_STATUS_NOT_STARTED);
-    cy.intercept('GET', '/v0/feature_toggles', {
+    cy.intercept('GET', '/v0/feature_toggles*', {
```

Comment:
I'm confused by this update. Why do some `intercept` URLs include a `**` prefix and this one doesn't? Also why isn't https://github.com/department-of-veterans-affairs/vets-website/pull/17131/files#diff-52b2fcb0444f109e051568e09ffbe1e77efae3baeb75371590f17e7e9064731bR73 getting modified at all??